pricing.js

```diff
@@ -1,37 +1,51 @@
+function formatPrice(price) {
+  return parseInt(price * 100) / 100
+}
+
+function discountPrice(product, price, dollarsOff) {
+  if (product.employerContribution.mode === 'dollar') {
+    price = price - product.employerContribution.contribution
+  } else {
+    dollarsOff = price * (product.employerContribution.contribution / 100)
+    price = price - dollarsOff
+  }
+  return price
+}
+
+function volLife(price, product, coverageLevels) {
```

Comment:
Remember, function names should always tell us what it is doing. calculateLTDPrice would be a much better name.